packages/config/config/devices/0x0346/alarm_keypad_gen2.json

```diff
@@ -19,302 +24,385 @@
 	"paramInformation": {
 		"1": {
 			"label": "Heartbeat Interval",
-			"description": "This parameter is the interval between heartbeats in minutes",
+			"description": "Number of minutes between heartbeats",
 			"valueSize": 1,
 			"unit": "minutes",
 			"minValue": 1,
 			"maxValue": 70,
 			"defaultValue": 70,
-			"unsigned": true,
 			"readOnly": false,
 			"writeOnly": false,
 			"allowManualEntry": true
 		},
 		"2": {
 			"label": "Application Retries",
-			"description": "This parameter is the number of application level retries attempted for messages",
+			"description": "Number of application level retries attempted for messages either not ACKed or messages encapsulated via supervision get that did not receive a report.",
 			"valueSize": 1,
 			"minValue": 0,
-			"maxValue": 1,
+			"maxValue": 5,
 			"defaultValue": 1,
-			"unsigned": true,
 			"readOnly": false,
 			"writeOnly": false,
 			"allowManualEntry": true
 		},
 		"3": {
 			"label": "Application Level Retry Base Wait Time Period",
-			"description": "This parameter is the base seconds used in the calculation for sleeping between retry messages",
+			"description": "The number base seconds used in the calculation for sleeping between retry messages.",
 			"valueSize": 1,
 			"unit": "seconds",
 			"minValue": 1,
-			"maxValue": 5,
+			"maxValue": 60,
 			"defaultValue": 5,
-			"unsigned": true,
 			"readOnly": false,
 			"writeOnly": false,
 			"allowManualEntry": true
 		},
 		"4": {
 			"label": "Announcement Audio Volume",
-			"description": "This parameter is the announce audio volume adjustment",
+			"description": "Announcement audio volume adjustment",
 			"valueSize": 1,
 			"minValue": 0,
-			"maxValue": 7,
+			"maxValue": 10,
 			"defaultValue": 7,
-			"unsigned": true,
 			"readOnly": false,
 			"writeOnly": false,
 			"allowManualEntry": true
 		},
 		"5": {
 			"label": "Key Tone Volume",
-			"description": "This parameter is the key tone volume adjustment",
+			"description": "Key tone volume adjustment",
 			"valueSize": 1,
 			"minValue": 0,
-			"maxValue": 6,
+			"maxValue": 10,
 			"defaultValue": 6,
-			"unsigned": true,
 			"readOnly": false,
 			"writeOnly": false,
 			"allowManualEntry": true
 		},
 		"6": {
 			"label": "Siren Volume",
-			"description": "This parameter is the siren volume adjustment",
+			"description": "Siren volume adjustment",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 10,
 			"defaultValue": 10,
-			"unsigned": true,
 			"readOnly": false,
 			"writeOnly": false,
 			"allowManualEntry": true
 		},
 		"7": {
 			"label": "Long Press Emergency Duration (seconds)",
```

Comment:
```suggestion
			"label": "Long Press Emergency Duration",
```

Reply:
What's the difference between this and param 8? If these reference specific buttons, I prefer labels like
`Emergency Button: Long Press Duration`
and
`Number Pad: Long Press Duration`

Reply:
Yes, that's it,  will change that.